nautobot/docs/rest-api/authentication.md

```diff
@@ -40,7 +40,7 @@ $ curl http://nautobot/api/dcim/sites/
 
 ## Initial Token Provisioning
 
-Ideally, each user should provision his or her own REST API token(s) via the web UI. However, you may encounter where a token must be created by a user via the REST API itself. Nautobot provides a special endpoint to provision tokens using a valid username and password combination.
+Ideally, each user should provision his or her own REST API token(s) via the web UI. However, you may encounter where a token must be created by a user via the REST API itself. Starting in version 1.3, Nautobot provides a special endpoint to provision tokens using a valid username and password combination.
```

Comment:
I like the thought here but I don't know if in the middle of a paragraph is best for visibility. We should really establish a general convention here that we can follow for all new feature documentation; personally I rather like the style used in the Python documentation, which would be something like:

```markdown
## Initial Token Provisioning via the REST API

_Added in version 1.3.0_

Ideally, each user should provision his or her own REST API token(s) via the web UI. However, you may encounter where a token must be created by a user via the REST API itself. Nautobot provides a special endpoint to provision tokens using a valid username and password combination.
```

Reply:
This is something we lack in a lot of places frankly. There is no mention of the version Secrets was introduced for example: https://nautobot.readthedocs.io/en/stable/core-functionality/secrets/